Command/CreateTweetContentTypeCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * File containing the CreateTweetContentType class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\TweetFieldTypeBundle\Command;
```

Comment:
Same comment than for `Command/CreateTweetContentCommand.php` (move to own commit).